test/executables/react16_upgrade_test.dart

```diff
@@ -71,25 +70,15 @@ ProcessResult runUpgrade({String onDirectory}) {
   return result;
 }
 
-ProcessResult runUpgradeWithFakeDartProject(
-    {PubspecCreator pubspecCreator, String mainDartContents}) {
-  var testPackage = DartTempProjectCreator(
-      pubspecCreator: pubspecCreator ?? tansitionPubspecCreator,
```

Comment:
`pubspecCreator` was always specified, meaning this default could be safely removed